packages/stats-py/aqsha_stats/analyses/sem_pls.py

```
@@ -0,0 +1,361 @@
+        tail = (proc.stderr or proc.stdout or "").strip()[-300:]
+        raise AnalysisError("pls_failed", f"Driver PLS-SEM gagal: {tail or 'output kosong'}")
+    error = payload.get("error")
+    if error:
+        raise AnalysisError(error.get("code", "pls_failed"), error.get("message", "PLS-SEM gagal."))
+    return payload
+
+
+def _rows(frame: dict) -> dict[str, dict[str, float | None]]:
+    """{"index_label": {column: value}} dari frame JSON driver."""
+    return {
+        idx: dict(zip(frame["columns"], row))
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)